includes/rewrites.php

```diff
@@ -5,6 +5,15 @@ function rewrite_location($location) {
     // FIXME -- also check the database for rewrites?
     global $config, $debug;
 
+    if (is_array($config['location_map_regex'])) {
+        foreach ($config['location_map_regex'] as $reg => $val) {
+            if (preg_match($reg, $location)) {
+                $location = $val;
+                continue;
```

Comment:
Makes so sense, this will keep the loop ongoing...

Break is needed, not continue.

A continue on last element is redundant anyways as the loop will continue regardless